src/main/java/com/epam/ta/reportportal/core/widget/content/filter/UniqueBugFilterStrategy.java

```diff
@@ -38,6 +44,19 @@
 	@Override
 	public Map<String, ?> buildFilterAndLoadContent(LoadContentStrategy loadContentStrategy, ReportPortalUser.ProjectDetails projectDetails,
 			Widget widget) {
-		return null;
+		UserFilter userFilter = widget.getFilter();
```

Comment:
this class is the same as ActivityFilterStrategy, so could we combine them in one common strategy? Something like ProjectFilterStrategy